scripts/install_custom_dc.sh

```diff
@@ -14,7 +14,12 @@
 # limitations under the License.
 set -e
 
-CUSTOM_DC_RELEASE_TAG=custom-dc-v0.3.2
+CUSTOM_DC_RELEASE_TAG=$1
+
+if [[ -z "$CUSTOM_DC_RELEASE_TAG" ]]; then
+    echo "Should run this script with first argument being the custom dc version tag found from https://github.com/datacommonsorg/website/tags"
```

Comment:
To confirm, is the whole thing the tag?  https://user-images.githubusercontent.com/4375037/233190347-baa90cbe-f080-482c-9a73-9846541fff53.png

Reply:
That's right.